### PR TITLE
set collection owner for collections created by CLI

### PR DESCRIPTION
**Pull request by tspewak**
Add owner to collection objects that are created with the CLI.

### CODE REVIEW

**Comment by tspewak** [comment]
Fixes #100

File: pkg/controller/collection/collection_controller.go [hunk 1]
```diff
@@ -191,6 +235,13 @@ func (r *ReconcileCollection) ReconcileCollection(c *kabanerov1alpha1.Collection
 	}
 	r_log = r_log.WithValues("Collection.Name", collectionName)
 
+	// A collection created by the CLI might not have kabanero as the owner.
+	// In that case we want to make kabanero the owner
+	err := r.ensureCollectionHasOwner(c, k)
+	if err != nil {
+		r_log.Error(err, "Could not make kabanero the owner of the collection")
```
Comment (kaczyns) [inline pkg/controller/collection/collection_controller.go]: Could we update the error message with the name of the collection (collectionName)?  I know we trace it just before, but I don't know if it's possible to configure the log to just print error messages, in which case we wouldn't have the collectionName.

Comment (tspewak) [reply pkg/controller/collection/collection_controller.go]: The r_log.WithValues("Collection.Name", collectionName) just above cause the collection name to come out on the r_log.Error(err, "Could not make kabanero the owner of the collection")

Comment (kaczyns) [reply pkg/controller/collection/collection_controller.go]: Thanks Tim.  I didn't understand how the log worked.  I will merge this.

**Pull request merged**